earth_enterprise/src/common/khxml/khdom.h

```diff
@@ -282,6 +289,18 @@ ToElementWithChildName(khxml::DOMElement *elem,
   }
 }
 
+
+template <class T>
+void
+ToElementWithChildName(khxml::DOMElement *elem,
+                       const std::string &childTagName,
+                       const MTVector<T> &val)
+{
+  auto theClosure = [&](const std::string& v) { AddElement(elem, childTagName, v); };
```

Comment:
Minor: I prefer writing lambdas inline in cases like this. I think it's easier to read.
```
  val.doForEach([&](const std::string& v) {
    AddElement(elem, childTagName, v);
  });
```

Reply:
The same comment applies pretty much everywhere you've used a function like this, though it's a stylistic issue so it's not a huge deal.